totalsize/total.py

```diff
@@ -139,7 +139,10 @@ def _readable_amount(self, amount, byte=False):
 
 
 class Playlist:
-    def __init__(self, url, format_sel, retries=0):
+    def __init__(self, url, format_sel, retries=0, cookies=None):
+        if cookies:
+            YTDL_OPTS['cookiefile'] = cookies
```

Comment:
Please make a new local variable, and then use that in the contruction of `self._ydl`.

Reply:
When you're attempting to modify a global variable inside a function without using global VAR, wouldn't it automatically become a local variable?

Reply:
Or would something like this be better?

Reply:
Exactly. Please use a lowercase name (something like `opts` is fine). All-caps names are for constants.

Reply:
```
opts = YTDL_OPTS
opts["cookiefile"] = cookies
```